woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -58,6 +58,14 @@ status_func() {
  return $1 #return with same status as entry param.
 }
 
+create_zram_swap() {
+  modprobe zram 2>/dev/null
```

Comment:
woops, should be this line
`modprobe zram 2>/dev/null || return 1`

Reply:
See my previous comment

Reply:
Ok, I'll remove the `|| return 1` from 2066bed